Documentation/CameraSystem/CreateSettingsProvider.md

```diff
@@ -3,7 +3,7 @@
 The Camera system is an extensible system for providing support for platform specific camera configurations. To add support for a new camera configuration, a custom settings provider may be required.
 
 > [!NOTE]
-> The complete source code used in this example can be found in the **MixedRealityToolkit.Extensions\Providers\Experimental\UnityAR** folder after importing the **Microsoft.MixedReality.Toolkit.Unity.Extensions** package.
+> The complete source code used in this example can be found in the **MRTK/Providers/UnityAR** folder.
```

Comment:
why remove the "requires extensions package" bit?

Reply:
it was incorrect data that should have been removed prior to shipping 2.2.0